torchtune/training/quantization.py

```diff
@@ -219,3 +223,22 @@ def swap_lora_linear_with_qat(
                 activation_qat_config,
                 weight_qat_config,
             )
+
+
+def convert_to_float8_training(model: torch.nn.Module) -> torch.nn.Module:
```

Comment:
nit: it's confusing to have `convert_to_float8_training` call `_convert_to_float8_training`, which then calls a different `convert_to_float8_training`.  Can we make the naming cleaner?